draft-ietf-quic-version-negotiation.md

```diff
@@ -80,13 +80,33 @@ which leverages similarities between versions and can negotiate between the set
 of "compatible" versions without additional round trips.
 
 
-## Conventions and Definitions
+## Conventions
 
 {::boilerplate bcp14-tagged}
 
-In this document, the Maximum Segment Lifetime (MSL) represents the time a QUIC
-packet can exist in the network. Implementations can make this configurable, and
-a RECOMMENDED value is one minute.
+
+## Definitions
+
+The document uses the following terms:
+
+* In the context of a given QUIC connection, the "first flight" of packets
+  refers to the set of packets the client creates and sends to initiate the
+  connection before it has heard back from the server.
+
+* In the context of a given QUIC connection, the "client's chosen version" is
+  the QUIC version of the connection's first flight.
+
+* The "original version" is the QUIC version of the very first packet the client
+  sends to the server. If version negotiation spans multiple connections, the
+  original version is equal to the client's chosen version of the first QUIC
+  connection.
+
+* The "negotiated version" is the QUIC version in use on the connection once the
+  version negotiation process completes.
+
+* The "Maximum Segment Lifetime" (MSL) represents the time a QUIC packet can
+  exist in the network. Implementations can make this configurable, and a
+  RECOMMENDED value is one minute.
```

Comment:
This is a little weird in that "segment" is a TCP concept.  You might need to acknowledge the origin of the term in order to explain that.